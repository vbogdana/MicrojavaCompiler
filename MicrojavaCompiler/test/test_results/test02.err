ERROR 05:57:37,718 - Semantics error at row 4 - type 'x' not found in symbol table
ERROR 05:57:37,718 - Syntax error at row 4 - unrecognized symbol ';'
ERROR 05:57:37,720 - Syntax error at row 7 - unrecognized symbol '='
ERROR 05:57:37,720 - Semantics error at row 8 - multiple definition of 'c'
ERROR 05:57:37,722 - Syntax error at row 10 - unrecognized symbol '='
ERROR 05:57:37,724 - Semantics error at row 13 - incompatible types
ERROR 05:57:37,724 - Semantics error at row 13 - expression must be a boolean value
ERROR 05:57:37,724 - Semantics error at row 14 - expression must be an integer value
ERROR 05:57:37,724 - Semantics error at row 14 - incompatible types
ERROR 05:57:37,724 - Syntax error at row 15 - unrecognized symbol '>'
ERROR 05:57:37,726 - Semantics error at row 17 - expression must be an integer value
ERROR 05:57:37,726 - Semantics error at row 17 - incompatible types
ERROR 05:57:37,726 - Semantics error at row 19 - symbol 'x' not found
ERROR 05:57:37,726 - Semantics error at row 19 - parameter must be an integer/character/boolean or a string value
ERROR 05:57:37,728 - Semantics error at row 20 - symbol 'x' not found
ERROR 05:57:37,728 - Semantics error at row 20 - expression must be an lvalue
ERROR 05:57:37,728 - Semantics error at row 21 - symbol 'max' not found
ERROR 05:57:37,728 - Semantics error at row 21 - parameter must be an integer/character/boolean or a string value
ERROR 05:57:37,728 - Parsing UNSUCCESSFUL!
