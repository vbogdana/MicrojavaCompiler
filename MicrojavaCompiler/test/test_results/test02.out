INFO  05:57:37,692 - Compiling source file: B:\STUDIJE\CETVRTA GODINA\sedmi semestar\PP\projekat\workspace\MicrojavaCompiler\test\test_files\test02.mj
INFO  05:57:37,718 - PARSING the program 'MaxCifra' at row 3
DEBUG 05:57:37,720 - Successful error recovery - constant definition
INFO  05:57:37,720 - Global variable 'a' declaration at row 6
INFO  05:57:37,720 - Global variable 'c' declaration at row 6
INFO  05:57:37,720 - Global variable 'maxcif' declaration at row 6
DEBUG 05:57:37,720 - Successful error recovery - global variable definition
INFO  05:57:37,722 - Function 'main' definition at row 10
INFO  05:57:37,722 - Local variable 'a' declaration at row 10
DEBUG 05:57:37,722 - Successful error recovery - local variable definition
INFO  05:57:37,724 - Use of Var a: char, 0, 1  at row 11
INFO  05:57:37,724 - Use of Var maxcif: int, 2, 0  at row 12
INFO  05:57:37,724 - Use of Var a: char, 0, 1  at row 13
INFO  05:57:37,724 - Use of Var c: int, 1, 0  at row 14
INFO  05:57:37,724 - Use of Var a: char, 0, 1  at row 14
INFO  05:57:37,724 - Use of Var maxcif: int, 2, 0  at row 15
DEBUG 05:57:37,726 - Successful error recovery - condition of an if/while construction
INFO  05:57:37,726 - Use of Var maxcif: int, 2, 0  at row 16
INFO  05:57:37,726 - Use of Var c: int, 1, 0  at row 16
INFO  05:57:37,726 - Use of Var a: char, 0, 1  at row 17
INFO  05:57:37,726 - Use of Var a: char, 0, 1  at row 17
INFO  05:57:37,728 - =======================SYNTAX ANALYSIS=======================
